pkg/common/si_helper_test.go

```diff
@@ -18,14 +18,21 @@
 package common
 
 import (
+	"strconv"
 	"testing"
 
+	"github.com/apache/incubator-yunikorn-k8shim/pkg/common/constants"
 	"github.com/apache/incubator-yunikorn-scheduler-interface/lib/go/common"
+	"github.com/apache/incubator-yunikorn-scheduler-interface/lib/go/si"
+
 	"gotest.tools/assert"
+
 	v1 "k8s.io/api/core/v1"
 	apis "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Can we re-order as we are cleaning up, 3 groups separated by an empty line:
- group system imports
- group other imports
- group yunikorn imports